jupyter_forward/core.py

```diff
@@ -83,8 +85,21 @@ def start(
     """
     password = getpass.getpass()
     session = Connection(host, connect_kwargs={'password': password})
+    logfile = f'~/.jforward.{port}'
```

Comment:
Can we put this in a user's `TMPDIR` instead of their home directory?

Reply:
That works for me -- is there an easy way to get remote environment variables, or should we do something like

```python
tmpdir = session.run('echo $TMPDIR`, hide=True).stdout
```

(which is in 9371c67)

Also, it turns out `hobart` doesn't define `$TMPDIR`, so there's a fallback to `$HOME` just in case.